### ISSUE TITLE
create download backup phrase file

### ISSUE THREAD

**Issue by guylepage3** [blockstack/blockstack-browser#576]
It would be much easier if there was a link the user could click to download their backup phrase. It would also be less friction for the user as they would not have to choose a backup phase filename.

<img width="1245" alt="screenshot 2017-06-29 19 03 46" src="https://user-images.githubusercontent.com/1711854/27714273-ef028da6-5cfd-11e7-80af-fb79e9bfa660.png">

**Comment by larrysalibra** [comment]
1password does a great job of this. It would make sense for us to do if the "secret key" on 1password's page was our encrypted identity key and and the blank space for master password was where the user wrote down their password. 

<img width="594" alt="screen shot 2017-08-20 at 4 18 13 pm" src="https://user-images.githubusercontent.com/597182/29493602-add1d71c-85c3-11e7-81b6-e20816bdaa09.png">

The problem with this approach for our situation is that it isn't really any better than only asking users to write down the identity key in terms of security.

Either way, whoever has this piece of paper has access to your account.

One benefit I see is that we could add instructions to the paper that the user might want to know in the case that they lost their account.

**Comment by vsund** [comment]
I guess that this file would very often stay somewhere on disk. Since the current situation is a paper backup, we could add a print feature instead of a download feature.

However in both cases we have to keep in mind that printing can be a security risk. Printers store printed files, files are sent through the network, and so on. Armory (if I remember correctly) has a way around this. They encrypt the backup with a short password (user choice vs random?) and there's a field where you can write it down on the printed backup.

**Comment by larrysalibra** [comment]
> However in both cases we have to keep in mind that printing can be a security risk. Printers store printed files, files are sent through the network, and so on. Armory (if I remember correctly) has a way around this. They encrypt the backup with a short password (user choice vs random?) and there's a field where you can write it down on the printed backup.

Yes! This is why we would have to put the encrypted phrase in the PDF and then leave a space for users to write (with a hand and pen) the password once they print off the document.

> I guess that this file would very often stay somewhere on disk. 
You're totally right! I never printed off my 1password "emergency kit" file so of course didn't write my master password in the blank box. I happen to have a printer at home but only really use it if forced to (usually by a government agency that wants a paper copy). Many (most?) people don't even have printers easily available these days. Instead I put my master password in my "real" password manager.

**Comment by guylepage3** [comment]
@larrysalibra.. Yes. I feel 1Password has done the best job with this for sure. I also have wondered if it's the most "secure" method though but I feel we severely reduce the risk of being hacked with your encrypted phrase solution. Really like this from a user's perspective.
